config/manager/deployment.yaml

```diff
@@ -52,9 +52,6 @@ spec:
         imagePullPolicy: Always
         name: manager
         resources:
-          limits:
-            cpu: 100m
-            memory: 500Mi
           requests:
             cpu: 100m
```

Comment:
Could you flip the requests to 10m as well, sounded like Derek is going to soon be asking for that as well so we may as well get ready.

Reply:
What the test does as of now is to check it it's present and if not, it says "recommend *min* value 10m". But I had the same questions. @derekwaynecarr can you check the two comments in this PR and confirm?

Reply:
a follow on test will validate that we schedule no more than 2 cores of cpu requests on master nodes.  Please set this to 10m.  The cpu time is relative to other components, and 10m is fine.

Reply:
basically, we will give the api server and etcd equal cpu time by making their request equal ~150m.  we will give the controller manager and scheduler less relative time ~100m.  any operator is basically a single controller in controller manager and giving it 1/10 what we will give controller manager is fine.  The key need is that we give operators matching cpu requests so they have a fair share of cpu time relative to each other, but less than etcd/APIServer

Reply:
Nice, thanks for clarifying @derekwaynecarr. PR updated.